packages/front-end/components/Experiment/UrlRedirectModal.tsx

```diff
@@ -365,36 +318,34 @@ const UrlRedirectModal: FC<{
         </div>
         <hr className="mt-4" />
         <div className="d-flex align-items-baseline my-1">
-          <input
-            type="checkbox"
-            {...form.register("persistQueryString")}
-            id={"toggle-persistQueryString"}
+          <Checkbox
+            label={
+              <div className="pr-2">
+                <label>Persist Query String</label>
+                <p style={{ color: "var(--color-text-mid)" }}>
+                  Allow user&apos;s queries, such as search terms, to carry over
+                  when redirecting
+                </p>
+              </div>
+            }
+            value={form.watch("persistQueryString")}
+            setValue={(v) => form.setValue("persistQueryString", v === true)}
           />
-          <label
-            htmlFor="toggle-persistQueryString"
-            className="text-muted ml-2"
-          >
-            <b>Persist Query String</b>
-            <p>
-              Keep this enabled to allow users’ queries, such as search terms,
-              to carry over when redirecting.
-            </p>
-          </label>
-          <input
-            type="checkbox"
-            {...form.register("circularDependencyCheck")}
-            id={"toggle-circularDependencyCheck"}
+          <Checkbox
+            label={
+              <>
+                <label>Circular Dependency Check</label>
+                <p style={{ color: "var(--color-text-mid)" }}>
```

Comment:
@bryce-fitzsimons @ahdriel - Still getting up to speed on the radix transition - is there a cleaner way to utilize our custom colors via Radix? 

(As an aside - I've got a question out to Kari to clarify if we always want the `description` of the `Checkbox` component to be this custom color. If so, I'll update the `Checkbox` component accordingly, but the question about is still valid).

Reply:
I think Radix has a `<Text>` component which we can use -- but I am unsure if we have updated its default style to use `color-text-mid`.

I do agree this should be a more broad change and it looks like our default text color should be `mid` (which is in line with your thoughts around updating the Checkbox component default text color).